docs/modules/ROOT/pages/integrating-with-existing-deployments.adoc

```diff
@@ -0,0 +1,100 @@
+= Integrating CMOS with an existing monitoring pipeline
+:icons: font
+:imagesdir: ../assets/images
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+
+[abstract]
+It is possible to integrate CMOS with existing monitoring pipelines.
```

Comment:
when loaded this is a box within a box (cant figure out how to put a picture in review) could this be changed?

Reply:
This is fine - might just be an artefact of how GitHub renders it, but I think it'll be fine in the published docs - lemme grab an example

Reply:
Hm, this is funky - look at https://github.com/couchbase/docs-server/blob/7311f0cccb80a9955d1ae036876d7cae2a12b0f6/modules/cli/pages/cbepctl/set-checkpoint_param.adoc, where the `abstract` is just an attribute substitution of the `description`

Reply:
This is how Antora renders it 
<img width="817" alt="image" src="https://user-images.githubusercontent.com/87416390/153849195-5a13b0a5-eb83-4114-bc21-3237c1efbd60.png">

Reply:
The outer block is for a title, see https://docs.asciidoctor.org/asciidoc/latest/sections/abstract-block/

Alternatively, it can be done as a section (the latest commit uses this) https://docs.asciidoctor.org/asciidoc/latest/sections/abstract/

Reply:
e.g., 
<img width="817" alt="image" src="https://user-images.githubusercontent.com/87416390/153849538-693d38ae-0bde-43ac-9070-a7ab5694de55.png">
and

Reply:
Abstract as a section (current):
<img width="817" alt="image" src="https://user-images.githubusercontent.com/87416390/153849783-f4c7326c-6253-4627-9932-7b25f9e8dd39.png">

Reply:
Hrm, ideally we'd stick to what the other CB docs are doing (e.g. https://raw.githubusercontent.com/couchbase/docs-server/7311f0cccb80a9955d1ae036876d7cae2a12b0f6/modules/cli/pages/cbepctl/set-checkpoint_param.adoc)

Reply:
It's finicky to get it to render in the way theirs does, and it makes me wonder if the concise abstract is even necessary / is repetitive:
![image](https://user-images.githubusercontent.com/87416390/153879735-a91f0abc-a3d9-4d84-a58d-7ecf58a067f2.png)

(Or maybe the title should be changed?..)

Reply:
Maybe, or maybe flesh out the abstract slightly?